src/main/java/com/google/sps/servlets/JoinRoomServlet.java

```diff
@@ -27,16 +27,15 @@
     to the associated private room once ensurred that it's an actual room*/
 @WebServlet("/join-room")
 public final class JoinRoomServlet extends HttpServlet {
-  private String inputtedUserTag = "user-party-link";
   private String currentRoomId = "";
```

Comment:
don't create a global variable here. Create a local variable inside the doGet function